internal/nodes/avi_model_nodes.go

```diff
@@ -270,6 +280,14 @@ func (o *AviObjectGraph) RemoveHTTPRefsFromSni(httpPol, hppMap string, sniNode *
 			}
 		}
 	}
+	for index, sgNode := range sniNode.StringGroupRefs {
+		for _, sgRef := range stringGroupToRemove {
+			if *sgNode.Name == sgRef {
```

Comment:
Same as above

Reply:
done

Reply:
The change is done and also visible in the Files changed.